x-pack/plugins/siem/public/components/link_to/link_to.tsx

```diff
@@ -23,6 +24,7 @@ export const LinkToPage = pure<LinkToPageProps>(({ match }) => (
     <Route path={`${match.url}/hosts/:hostName`} component={RedirectToHostsPage} />
     <Route exact path={`${match.url}/network`} component={RedirectToNetworkPage} />
     <Route path={`${match.url}/network/ip/:ip`} component={RedirectToNetworkPage} />
+    <Route path={`${match.url}/timelines`} component={RedirectToTimelinesPage} />
```

Comment:
Adding a trailing slash results in the below. There's an open issue for this already (https://github.com/elastic/ingest-dev/issues/410), so I'll add a comment for the Timeline header as well.

![image](https://user-images.githubusercontent.com/2946766/57030419-d670e200-6c01-11e9-888c-fa33ccdd83ed.png)